bench/bm_case1.cpp

```diff
@@ -7,44 +6,128 @@
 
 #include "bm_test_helper.hpp"
 #include <graph.hpp>
+#include <vir/simd.h>
 
-// #define RUN_SIMD_TESTS
+namespace fg = fair::graph;
 
-namespace fg                           = fair::graph;
-
-inline constexpr std::size_t N_ITER    = 10;
+inline constexpr std::size_t N_ITER = 10;
 inline constexpr std::size_t N_SAMPLES = gr::util::round_up(1'000'000, 1024);
 
 template<typename T, int addend, std::size_t N_MIN = 0, std::size_t N_MAX = N_MAX>
-class add : public fg::node<add<T, addend, N_MIN, N_MAX>, fg::IN<T, "in", N_MIN, N_MAX>, fg::OUT<T, "out", N_MIN, N_MAX>, fg::limits<N_MIN, N_MAX>> {
+class add
```

Comment:
can the formatting changes be skipped in the patch - will produce a merge issue with the other patch

Reply:
I do not want to fight clang-format.... we need to iterate (sooner) on how we want to write/format code automatically.

Reply:
FTW, my clang-format apparently fight yours. I.e. it reverts this formatting change.